packages/mdc-textfield/mdc-textfield.scss

```diff
@@ -242,57 +287,6 @@ $mdc-textfield-disabled-border-on-dark: rgba(white, .3);
     }
   }
 
-  // TODO: Move into general .mdc-textfield decl and replace ::after textfield styles with this
-  // element as part of #675
-  .mdc-textfield__bottom-line {
-    position: absolute;
-    bottom: 0;
-    left: 0;
-    width: 100%;
-    height: 1px;
-    transform: scaleY(1);
-    transform-origin: center bottom;
-    transition:
-      background-color 33ms $mdc-animation-standard-curve-timing-function,
-      mdc-textfield-transition(transform);
-    background-color: rgba(black, .42);
-    pointer-events: none;
-
-    &::after {
```

Comment:
I'm a little confused what happened to these `::after` styles?

Reply:
It became a real boy